python/lvmdrp/functions/run_drp.py

```diff
@@ -956,6 +961,110 @@ def combine_cameras(tileid: int, mjd: int, spec: int = 1):
         log.info(f'Output combined camera file: {bout_file}')
 
 
+def combine_spectrographs(tileid: int, mjd: int, expnum: int) -> fits.HDUList:
+    """ Combine the spectrographs together for a given exposure
+
+    For a given exposure, combines the three spectographs together
+    into a single output lvm-CFrame file.  The input files are the
+    ancillary camera-combined lvm-object-sp[id]-[expnum] files.
+
+    Parameters
+    ----------
+    tileid : int
+        The tileid of the observation
+    mjd : int
+        The MJD of the observation
+    expnum : int
+        The exposure number of the frames to combines
+
+    Returns
+    -------
+    fits.HDUList
+        the output FITS file
+    """
+
+    files = sorted(path.expand('lvm_anc', mjd=mjd, tileid=tileid, drpver=drpver,
+                               kind='', camera='sp*', imagetype='object', expnum=expnum))
+
+    if not files:
+        log.error(f'No camera-combined files found for expnum: {expnum}')
+        return
+
+    if len(files) != 3:
+        log.warning(f'Warning: Not all specids found for expnum: {expnum}')
+
+    # construct output path
+    cframe = path.full('lvm_frame', mjd=mjd, tileid=tileid, drpver=drpver,
+                       kind='CFrame', expnum=expnum)
+
+    # get the first header
+    with fits.open(files[0]) as hdu:
+        hdr = hdu[0].header.copy()
+
+    # build the wavelength axis
+    wcs = WCS(hdr)
+    n_wave = hdr['NAXIS1']
+    wl = wcs.spectral.all_pix2world(np.arange(n_wave), 0)[0]
+    wave = fits.ImageHDU((wl * u.m).to(u.angstrom).value, name='WAVE')
+
+    # get total number of fibers from the fibermap
+    # do we use this to check the total output fiber number? should be the same?
+    total_fibers = len(fibermap.data)
+
+    # stack the data in the extensions
+    flux_data = stack_ext(files, ext=0)
+    err_data = stack_ext(files, ext='ERROR')
+    mask_data = stack_ext(files, ext='BADPIX')
+    fwhm_data = stack_ext(files, ext='INSTFWHM')
+
+    # update the primary header
+    hdr['SPEC'] = ', '.join([i.split('-')[2] for i in files])
+    hdr['FILENAME'] = pathlib.Path(cframe).name
+    hdr['VERSDRP'] = drpver
```

Comment:
Why not `DRPVER`? Seems more memorable and consistent with the naming conventions in the code.

Reply:
I can rename it to DRPVER.  I just used the original syntax for MaNGA.